sdk/python/lib/pulumi/dynamic/__main__.py

```diff
@@ -210,6 +245,12 @@ def Read(self, request, context):
     def __init__(self):
         pass
 
+    def _get_config_arg(self, method: Callable) -> Optional[Config]:
+        sig = signature(method)
+        if len(sig.parameters) == 2 and sig.parameters.get("config") is not None:
```

Comment:
What is the purpose of the `== 2` check here? If it's checking the number of arguments, does this mean `check` would fail for instance (since it takes 3 with config included)? If it's not checking the number of arguments, what is it checking? I think a comment could be helpful here either way.

Reply:
Ooops, yea the `== 2` works for create, which has 1 or 2 arguments (props and optionally config), but other methods take more. I think we can just check for `config` instead.